server_manager/www/app.ts

```diff
@@ -1035,30 +1035,52 @@ export class App {
     }
   }
 
-  private async refreshTransferStats(
+  private async refreshServerMetrics(
     selectedServer: server_model.Server,
     serverView: ServerView
   ) {
     try {
-      const usageMap = await selectedServer.getDataUsage();
-      const keyTransfers = [...usageMap.values()];
+      const serverMetrics = await selectedServer.getServerMetrics();
+
+      let totalUserHours = 0;
+      for (const {tunnelTime} of serverMetrics.server) {
+        // convert to hours
```

Comment:
nit: Kind of a pointless comment tbh given the naming of the variables. If it is unclear, prefer to make 60*60 self-documenting, then you won't need the comment. Something like:

```ts
const HOUR_IN_SECS = 60 * 60;

...

totalUserHours += tunnelTime.seconds / HOUR_IN_SECS;
```